tests/platform/windows/windows-zoomtext-testSpec.txt

```diff
@@ -0,0 +1,9 @@
+windows-zoomtext-testSpec.js
+
+Descriptions:
+This will run the acceptance tests for the ZoomText 10.1 (AiSquared). It uses 4 NP sets: zoomtext_application.json, zoomtext_common.json, zoomtext_common2.json and zoomtext_common3.json
```

Comment:
All these prefSets are missing, please do add them.

Reply:
I added one common preference set, since it's the only one from the old ones that is compatible with new ZoomText settings.